src/features/prompts/components/promote-prompt.tsx

```diff
@@ -15,21 +15,21 @@ export function PromotePrompt({
   projectId,
   promptName,
   disabled,
+  variant,
 }: {
   promptId: string;
   projectId: string;
   promptName: string;
   disabled: boolean;
+  variant?: "ghost" | "outline";
 }) {
-  const [isPromoted, setIsPromoted] = useState(false);
   const utils = api.useUtils();
   const posthog = usePostHog();
-
+  const [isOpen, setIsOpen] = useState(false);
```

Comment:
Did the popover not close automatically?

Reply:
No, it didn't. In the Prompts table it worked as we update the underlying data in the table creating a refresh - reminds me of the tag popover 😄 
In the details view it is part of the toolbox so we needed an logic for it to close. See screencast in Linear for the behavior without the logic.